src/arrays.jl

```diff
@@ -1,10 +1,12 @@
-function pm_Array{T}(vec::AbstractVector) where T
+const pm_Array_suppT = Union{Int32, Int64, String, pm_Set{Int32}, pm_Matrix{pm_Integer}, AbstractString}
```

Comment:
This has to be changed for each new small type we bind, right? This looks like it will introduce bugs at some point :)

Reply:
this is just to make the signatures below shorter and to have just one place to be updated once a new element type is added. What do You suggest

Reply:
I thought these should match the types listed [here](https://github.com/oscar-system/Polymake.jl/blob/6ae6fb7026debab304db837c6b6a51b32c37d7ee/deps/type_setup.jl#L14-L23) or is this just for a subset?

Reply:
that's right. Those are wrapped, but never used (and obviously never tested).
We could add support on to construct them on the julia side if we wish (you can interact with them if polymake returns them, just fine), but I think it's not particularly necessary (until some voices its demand).

And no one is willing to add tests for them anyway :P